sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/TypeCoercion.scala

```diff
@@ -846,6 +848,12 @@ object TypeCoercion {
         Cast(TimeAdd(l, r), l.dataType)
       case Subtract(l, r @ CalendarIntervalType()) if acceptedTypes.contains(l.dataType) =>
         Cast(TimeSub(l, r), l.dataType)
+      case Multiply(l @ CalendarIntervalType(), r @ NumericType()) =>
+        MultiplyInterval(l, r)
+      case Multiply(l @ NumericType(), r @ CalendarIntervalType()) =>
+        MultiplyInterval(r, l)
+      case Divide(l @ CalendarIntervalType(), r @ NumericType()) =>
```

Comment:
```sql
postgres=# select interval '1 year' / '365';
   ?column?
---------------
 23:40:16.4064
(1 row)
```
could this be supported?

Reply:
Taking into account the discussion in https://github.com/apache/spark/pull/26165, I am not sure. @cloud-fan Should I support this?

Reply:
We can, but this should only apply to literals, not string columns.

Reply:
If we want to support it, please open another PR.